logger.go

```diff
@@ -103,7 +103,7 @@ func LoggerWithWriter(out io.Writer, notlogged ...string) HandlerFunc {
 				path = path + "?" + raw
 			}
 
-			fmt.Fprintf(out, "[GIN] %v |%s %3d %s| %13v | %15s |%s %-7s %s %s\n%s",
+			fmt.Fprintf(out, "[GIN] %v |%s %3d %s| %13v | %39s |%s %-7s %s %s\n%s",
 				end.Format("2006/01/02 - 15:04:05"),
```

Comment:
check the client ip length first. Please try the following example.

```go
width := 0
if len(clientIP) > 15 {
  width = 39
} else {
  width = 15
}

fmt.Printf("%*s\n", width, "Hello World!")
```

see https://play.golang.org/p/pLKiETJ-NX

Reply:
```go
width := 15
if len(clientIP) > 15 {
  width = 39
}
```